DirectEmployers/MyJobs
# [PD-3369] refactor analytics reducer

## What's this PR do?
Refactors the analytics reducers

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application

## How should this be manually tested?
Pull down the repository. Go into analytics and make sure all of the actions associated with clicking on anything still works

## Any background context you want to provide?
More code based than visual based so looking at the code would be better
